api/server/call_logs.go

```diff
@@ -21,18 +27,19 @@ func (s *Server) handleCallLogGet(c *gin.Context) {
 		return
 	}
 
-	// TODO this API needs to change to text/plain / gzip anyway, punting
-	// optimization, but we can write this direct to the wire, too... seems like
-	// we should write some kind of writev json thing for go since we keep
-	// hitting this :(
 	var b bytes.Buffer
 	b.ReadFrom(logReader)
 
-	callObj := models.CallLog{
-		CallID:  callID,
-		AppName: appName,
-		Log:     b.String(),
+	switch c.Request.Header.Get("Accept") {
+	case "application/json":
```

Comment:
this only gets the first value from Accept, which may be a list, and we likely want to prefer plaintext

Reply:
I do aware that Accept can be a list, but fn_go sends exactly one item (which is `application/json`). So the best option i see here is to loop over MIME types.

Reply:
ok, for the old endpoint that makes sense. do the updated swagger clients request `text/plain` properly for this endpoint?